### PR TITLE
ShuffleChannels - rename attribute groups->group

### PR DESCRIPTION
**Pull request by postrational**
Rename attribute groups->group to match specification.

### CODE REVIEW

File: src/ngraph/op/fused/shuffle_channels.cpp [hunk 1]
```diff
@@ -23,18 +23,18 @@ using namespace ngraph;
 
 constexpr NodeTypeInfo op::ShuffleChannels::type_info;
 
-op::ShuffleChannels::ShuffleChannels(const Output<Node>& data, const int axis, const size_t groups)
+op::ShuffleChannels::ShuffleChannels(const Output<Node>& data, const int axis, const size_t group)
```
Comment (diyessi) [inline src/ngraph/op/fused/shuffle_channels.cpp:26]: Noted in passing:
Would it be a problem to make `groups` a `uint64_t`? The size of `size_t` varies from platform to platform.
Same with `axis` -- it should be `int64_t`

Comment (rkimballn1) [reply src/ngraph/op/fused/shuffle_channels.cpp:26]: There is nothing wrong with size_t but there is a problem when people assume that size_t is 64 bits. You should never mix size_t with other types like int64_t, if you use size_t use size_t everywhere it touches.

Comment (diyessi) [reply src/ngraph/op/fused/shuffle_channels.cpp:26]: People assume that a lot of places, and the compiler isn't warning about it when it's not true.

**Comment by lazarevevgeny** [comment]
The change in the DLDT repo will be done as part of MR #7528

**Pull request merged**